apps/examples/testcase/Makefile

```diff
@@ -80,6 +80,7 @@ include le_tc/filesystem/Make.defs
 include le_tc/kernel/Make.defs
 include le_tc/network/Make.defs
 include le_tc/ttrace/Make.defs
+include le_tc/drivers/Make.defs
```

Comment:
alphabetic order ?

Reply:
Is this present in any guidelines?
Alphabetic order has not been followed at several places. Even in the same file.

Reply:
Fixed this also